parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -518,6 +518,12 @@ configGPUDrivers() {
 }
 
 validateGPUDrivers() {
+    CPU_ARCH=$(getCPUArch)  #amd64 or arm64
```

Comment:
nit: L521-L526 are same as L547-L552. worth moving that logic to a helper function since getCPUArch is already one? if not, cool as is.

Reply:
thanks for the comments. I added a new function isARM64() for this purpose.